packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -412,15 +449,136 @@ describe('AppSec Index', () => {
           'content-type': 'application/json',
           'content-lenght': 42
         },
-        'server.request.body': { a: '1' },
         'server.request.query': { b: '2' },
         'server.request.framework_endpoint': '/path/:c',
-        'server.request.path_params': { c: '3' },
-        'server.request.cookies': { d: [ '4' ], e: [ '5' ] }
+        'server.request.path_params': { c: '3' }
       }, context)
       expect(Reporter.finishRequest).to.have.been.calledOnceWithExactly(req, context)
     })
   })
+
+  describe('checkRequestData', () => {
```

Comment:
When the waf is updated and we have whole information, we should add more plugin style tests, to execute real express/body-parser/cookie-parser and do request that will be blocked or not.